core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -205,26 +205,38 @@ private void populateValues(HttpServletRequest request) throws IOException {
     String ufsRoot = Configuration.get(PropertyKey.MASTER_MOUNT_TABLE_ROOT_UFS);
     UnderFileSystem ufs = UnderFileSystem.Factory.create(ufsRoot);
 
-    long sizeBytes = ufs.getSpace(ufsRoot, UnderFileSystem.SpaceType.SPACE_TOTAL);
-    if (sizeBytes >= 0) {
-      request.setAttribute("diskCapacity", FormatUtils.getSizeFromBytes(sizeBytes));
-    } else {
-      request.setAttribute("diskCapacity", "UNKNOWN");
+    String totalSpace = "UNKNOWN";
+    try {
+      long sizeBytes = ufs.getSpace(ufsRoot, UnderFileSystem.SpaceType.SPACE_TOTAL);
+      if (sizeBytes >= 0) {
+        totalSpace = FormatUtils.getSizeFromBytes(sizeBytes);
+      }
+    } catch (IOException e) {
+      // Exception may be thrown when UFS connection is lost
```

Comment:
do you think it's worth putting WARN logging here?

Reply:
fixed